docs/serverless-v2/06-01-function-cr.md

```diff
@@ -52,12 +76,42 @@ This table lists all the possible properties of a given resource together with t
 |----------|:---------:|-------------|
 | **metadata.name** | Yes | Specifies the name of the CR. |
 | **metadata.namespace** | No | Defines the Namespace in which the CR is available. It is set to `default` unless you specify otherwise. |
-| **spec.functionContentType** | No | Specifies the content type of the lambda's code defined in the **function** property. The content type can be either `plaintext` or `base64`. It is set to `plaintext` unless you specify otherwise.|
-| **spec.runtime** | No | Specifies the software runtime used to run the lambda's code. The Function Controller supports `nodejs6` and `nodejs8`. It is set to `nodejs8` unless you specify otherwise. |
 | **spec.timeout** | No | Specifies the duration in seconds after which the lambda execution is terminated. The default value is `180`. |
 | **spec.env** | No | Specifies environment variables you need to export for the lambda. |
 | **spec.deps** | No | Specifies the lambda's dependencies. |
-| **spec.function** | Yes | Provides the lambda's source code. |
+| **spec.minReplicas** | No | Defines the minimum number of lambda's Pods to run at a time. |
+| **spec.maxReplicas** | No | Defines the maximum number of lambda's Pods to run at a time. |
+| **spec.resources.limits.cpu** | No | Maximum number of CPUs available for the lambda's Pod to use. |
+| **spec.resources.limits.memory** | No | Maximum amount of memory available for the lambda's Pod to use. |
+| **spec.resources.requests.cpu** | No |  Number of CPUs requested by the lambda's Pod to operate. |
+| **spec.resources.requests.memory** | No | Amount of memory requested by the lambda's Pod to operate. |
+| **spec.source** | Yes | Provides the lambda's source code. |
+| **status.conditions.lastTransitionTime** | Not applicable | Provides a timestamp for the last time the lambda's Pod transitioned from one condition to another. |
```

Comment:
Not lambda pod, last time that the condition changed the status

Reply:
condition type, right? to be consistent with the terminology we introduced